litmus-portal/frontend/src/components/ProgressBar/AnalyticsLinearProgressBar/index.tsx

```diff
@@ -1,29 +1,97 @@
-import { Line } from 'rc-progress';
 import React from 'react';
-import { useTheme } from '@material-ui/core/styles';
+import { withStyles, lighten, LinearProgress, Theme } from '@material-ui/core';
 
 interface LinearProgressBarProps {
   value: number;
   maxValue: number;
   isInTable: boolean;
 }
+const lightConstant = 0.5;
+
+const RedLinearProgress = withStyles((theme: Theme) => ({
+  bar: {
+    backgroundColor: theme.palette.error.dark,
+  },
+  root: {
+    backgroundColor: lighten(theme.palette.error.dark, lightConstant),
+  },
+}))(LinearProgress);
+
+const YellowLinearProgress = withStyles((theme: Theme) => ({
+  bar: {
+    backgroundColor: theme.palette.warning.main,
+  },
+  root: {
+    backgroundColor: lighten(theme.palette.warning.main, lightConstant),
+  },
+}))(LinearProgress);
+
+const GreenLinearProgress = withStyles((theme: Theme) => ({
+  bar: {
+    backgroundColor: theme.palette.primary.dark,
+  },
+  root: {
+    backgroundColor: lighten(theme.palette.primary.dark, lightConstant),
+  },
+}))(LinearProgress);
```

Comment:
We already have a `LinearProgressBar` component, you can modify that a bit in order to accommodate the color changes. You don't need to create Progress Bars like this here.